src/setup.py

```diff
@@ -29,29 +30,32 @@
 version of Python. If you can't upgrade your pip (or Python), request
 an older version of pretix:
     $ python -m pip install "pretix<2"
-""".format(*(REQUIRED_PYTHON + CURRENT_PYTHON)))
+""".format(
+            *(REQUIRED_PYTHON + CURRENT_PYTHON)
+        )
+    )
     sys.exit(1)
 
 here = path.abspath(path.dirname(__file__))
 npm_installed = False
 
 # Get the long description from the relevant file
 try:
-    with open(path.join(here, '../README.rst'), encoding='utf-8') as f:
+    with open(path.join(here, "../README.rst"), encoding="utf-8") as f:
```

Comment:
Why do you include unrelated changes here?

Reply:
my flaker must have kicked in when i saved the file, should i revert this?

Reply:
yes, please revert unrelated changes.

Reply:
Yes, please revert all unrelated changes. Thanks